man/systemd.special.xml

```diff
@@ -240,6 +240,9 @@
             names like <varname>single</varname>, <varname>rescue</varname>, <varname>1</varname>,
             <varname>3</varname>, <varname>5</varname>, …; see
             <citerefentry><refentrytitle>systemd</refentrytitle><manvolnum>1</manvolnum></citerefentry>.</para>
+
+            <para>Please note that using WantedBy=default.target for normal services is not a good idea,
```

Comment:
write `<literal>WantedBy=default.target</literal>`

maybe say "regular" rather than "normal".

And maybe add "typically" or so, to tone it down a bit.

i.e. "Please note that using … for regular services is typically not a good idea, …"